src/z3c/currency/converter.py

```diff
@@ -20,10 +20,10 @@
 from z3c.currency import interfaces
 from zope.i18n import format
 
+@zope.interface.implementer(IDataConverter)
 class CurrencyConverter(object):
     """Converts currency fields to text representations."""
     zope.component.adapts(interfaces.ICurrency, IWidget)
-    zope.interface.implements(IDataConverter)
```

Comment:
Yes, absolutely. It seems like I never got around to writing fixers for that. My bad. Will fix manually for now.